src/applications/disability-benefits/2346/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { createSaveInProgressFormReducer } from 'platform/forms/save-in-progress/reducers';
+import formConfig from '../config/form';
+import form2346Reducer from './form2346Reducer';
+
+export default {
+  form: createSaveInProgressFormReducer(formConfig),
```

Comment:
Is this used? Or is it just the `form2346Reducer`?

Reply:
@kelsonic We are using it in our components. Look at our `mapStateToProps`

Reply:
I just checked and you are not using `state.form` anywhere, so I think you can omit line 6 here @mojaray2k

Reply:
@kelsonic Can't get rid of that man. That is for the Save In Progress Widget that is provided by the VA team.

Reply:
Ah alrighty! Thanks for confirming 👍